php/commands/shell.php

```diff
@@ -35,7 +35,7 @@ public function __invoke( $_, $assoc_args ) {
 		if ( '\\Psy\\Shell' == $class ) {
 			\Psy\Shell::debug();
 		} else {
-			$repl = new $class( 'wp> ' );
+			$repl = new $class(  "\nwp> " );
```

Comment:
> Happy to open another issue/PR for that if you think it's worth the tinker?

Yes please. We should also look into whether we can inject the newline after command execution.